insights/parsers/dmidecode.py

```diff
@@ -110,7 +110,8 @@ class DMIDecode(Parser, LegacyItemAccess):
     """
     Class for DMI information.
     """
-
+    # TODO:
+    # Remove below 4 dicts
```

Comment:
This comment is not very clear about when and why to remove.  Please add some context here to explain to another developer why this TODO is here.